hacks/ops-pod

```diff
@@ -49,6 +50,7 @@ image=
 tolerations_array="
   - operator: Exists
 "
+hostnetwork="false"
```

Comment:
```suggestion
hostnetwork=${FALSE}

Reply:
FALSE variable is a number, however the value of `hostnetwork` will then be used to set `spec.template.spec.hostNetwork`, which expects a `bool` value -> we can't do this change

Reply:
Indeed, I tested it like this before and it didn't work...

I could add like an IF to check for that variable and then assign the associated string, but I think that it's easier and simpler (thus better) to just use the string values directly.

Reply:
Let's leave it as it is. Was a bit distracted when I did the review sorry.